app/src/app/components/routes/ProtectedRoute.tsx

```diff
@@ -18,15 +19,29 @@ export default function ProtectedRoute() {
     useEffect(() => {
         const user = getCurrentUser();
 
-        if (authConfig.enableAuth) {
-            if (!isAuthenticated && !user?.token) {
+        if (!authConfig.enableAuth) {
+            return;
+        }
+
+        if (!isAuthenticated && !user?.token) {
+            navigate("/login");
```

Comment:
Just for the sake of DRY, would be nice to pull out this line as it gets done in two places